src/test/java/com/commercetools/project/sync/CliRunnerTest.java

```diff
@@ -272,6 +273,32 @@ void run_AsCartDiscountFullSync_ShouldBuildSyncerAndExecuteSync() {
     verify(syncerFactory, never()).syncAll(null, true);
   }
 
+  @Test
+  void run_AsStateFullSync_ShouldBuildSyncerAndExecuteSync() {
```

Comment:
If this is for `StateFullSync` what about the `StateDeltaSync` test case?

Reply:
I think  this test is redundant because we already tested in an integration tests these stuff. I have to add this redundant test because of the coverage :) If you look at the other resources, they also have only full sync tests.